slack-api-client/src/test/java/test_with_remote_apis/methods_admin_api/AdminApi_apps_Test.java

```diff
@@ -57,14 +57,14 @@ public void apps_uninstall() throws Exception {
     public void apps_approvedList() throws Exception {
         if (orgAdminUserToken != null) {
             AdminAppsApprovedListResponse errorResponse = methodsAsync.adminAppsApprovedList(r -> r
-                            .limit(1000)
-                    /*.teamId(adminAppsTeamId)*/)
+                                    .limit(1000)
+                            /*.teamId(adminAppsTeamId)*/)
```

Comment:
Was this indentation intentional? It looks misaligned compared to the rest of the file, but that could be GitHub's editor.

Reply:
The latest IDEA editor’s code formatter made this change. A bit weird, though.